data/json/items/comestibles/med.json

```diff
@@ -0,0 +1,251 @@
+        "addiction_potential" : 15,
+        "nutrition" : 0,
+        "description" : "Cough syrup made from mutated poppy.  Will make you sleepy.",
+        "price" : 2500,
+        "volume" : 1,
+        "phase" : "liquid",
+        "charges" : 5,
+        "fun" : 0
+    },
+    {
+        "type" : "COMESTIBLE",
+        "id" : "poppy_sleep",
+        "name" : "poppy sleep",
+        "weight" : 1,
+        "color" : "magenta",
+        "addiction_type" : "opiate",
+        "use_action" : "SLEEP",
+        "comestible_type" : "MED",
+        "symbol" : "!",
+        "quench" : 0,
+        "healthy" : -2,
+        "nutrition" : 0,
+        "description" : "A potent sleep aid extracted from mutated poppy seeds.  Effective, but as an opiate, it may be addictive.",
+        "price" : 2000,
+        "material" : "powder",
+        "volume" : 1,
+        "charges" : 10,
+        "stack_size" : 100,
+        "fun" : 0
+    },
+    {
+        "type" : "COMESTIBLE",
+        "id" : "poppy_pain",
+        "name" : "poppy painkiller",
+        "weight" : 1,
+        "color" : "pink",
+        "addiction_type" : "opiate",
+        "use_action" : {
+            "type" : "consume_drug",
+            "activation_message" : "You take some poppy painkiller.",
+            "effects" : [
+                {
+                    "id" : "pkill2",
+                    "duration" : 180
+                }
+            ]
+        },
+        "comestible_type" : "MED",
+        "symbol" : "!",
+        "quench" : 0,
+        "healthy" : -2,
+        "addiction_potential" : 10,
+        "nutrition" : 0,
+        "description" : "Potent opioid palliative produced by the refining of the mutated poppy.  Notably devoid of euphoric or sedative effects, as an opiate it may still be addictive.",
+        "price" : 3000,
+        "material" : "powder",
+        "volume" : 1,
+        "charges" : 10,
+        "stack_size" : 100,
+        "fun" : 0
+    },
+    {   "type" : "COMESTIBLE",
+        "id" : "datura_seed",
+        "name" : "datura seeds",
+        "name_plural" : "datura seeds",
+        "category" : "other",
+        "weight" : 2,
+        "color" : "brown",
+        "use_action" : "DATURA",
+        "stim" : 6,
+        "comestible_type" : "MED",
+        "symbol" : ".",
+        "quench" : 0,
+        "healthy" : -2,
+        "nutrition" : 0,
+        "description" : "Small, dark seeds from the spiny pods of a datura plant.  Full of powerful psychoactive chemicals, these tiny seeds are a potent analgesic and deliriant, and can be deadly in cases of overdose.  You could probably plant these.",
+        "price" : 0,
+        "volume" : 1,
+        "charges" : 4,
+        "stack_size" : 100,
+        "fun" : -1,
+        "seed_data" : {
+            "plant_name" : "datura",
+            "fruit" : "datura_seed",
+            "byproducts" : [ "withered" ],
+            "grow" : 91
+        }
+    },
+    {
+        "type" : "COMESTIBLE",
+        "id" : "thyme_oil",
```

Comment:
`id` first, then `type`, then `name`, then `description`. After that the ordering is up to you but try to make each entry consistent